### DISCUSSION TITLE
redirects to INSPIRE knowledge base

### DISCUSSION THREAD

**Discussion by JanKoers** [INSPIRE-MIF/helpdesk#194] [Q&A] [answered]
We are continually updating our national INSPIRE documents. Some of the links that are present in that documents refer to old URL’s but are automatically redirected to the new URL’s. What are the steps we should take according to your policies: leave the URL’s as they are because there are active redirects, or change the URL’s to the new URL’s?

**Comment by fabiovinci** [comment, marked as answer]
Dear @JanKoers,

some links serve as unique persistent identifiers for related documents and will be maintained in the future.
E.g., all links starting with http://inspire.ec.europa.eu/documents/ redirect to their relevant pages in the new IKB.

For this reason, the use of links that act as identifiers is recommended.
